backend/src/schema/resolvers/searches.js

```diff
@@ -20,7 +20,7 @@ export default {
       AND NOT (
         author.deleted = true OR author.disabled = true
         OR resource.deleted = true OR resource.disabled = true
-        OR (:User { id: $thisUserId })-[:BLOCKED]-(author)
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T17:29:40Z" title="Thursday, January 23rd 2020, 6:29:40 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -20,7 +20,6 @@ export default {
       AND NOT (
         author.deleted = true OR author.disabled = true
         OR resource.deleted = true OR resource.disabled = true
-        OR (:User { id: $thisUserId })-[:BLOCKED]-(author)
```


as discussed in our pairing session @roschaefer ... this was unintended after merging in changes from `master`
I found out why it was still passing the cypress tests and will push up the code...
turns out we had some issues with copy :spaghetti: in the `Search User` PR where we didn't properly rename variables